web/src/user/LibraryPage/ApplicationList.ts

```diff
@@ -0,0 +1,96 @@
+import type { AppGroupEntry } from "./types.js";
```

Comment:
This is largely ported from `ak-application-list` as function component, making a lot of the focus logic possible when connecting search inputs to their respective cards.